lib/mongodb.js

```diff
@@ -1870,30 +1873,71 @@ MongoDB.prototype.ping = function(cb) {
   }
 };
 
+// Determine if a property must have an ObjectID value
+function mustBeObjectID(propDef) {
```

Comment:
Why `must`? The name implies to me that an error is thrown when `propDef` is not ObjectID`. Proposed name:

```suggestion
function isStoredAsObjectID(propDef) {
```

Reply:
You are correct, if `mustBeObjectID(propDef) === true`, an error will be thrown by `coerceToObjectId()` if the value is not an `ObjectID`.

`isStoredAsObjectID` does not convey the intended message, IMO.

Reply:
Well if the error is being thrown by `coerceToObjectId()`, I think it makes sense to rename this function to something like `isObjectIDType` (fine with `isStoredAsObjectID`) since objectively looking at this function alone, it's used to determine if the data type is `ObjectID`.

Reply:
Renamed to `isStoredAsObjectID `.